### PR TITLE
DROOLS-4407: [DMN Designer] Search feature - Highlight the current result with colors that follow the PF standard

### PR DESCRIPTION
**Pull request by danielzhe**
This PR is part of an ensemble and also covers [DROOLS-4406](https://issues.jboss.org/projects/DROOLS/issues/DROOLS-4406).

https://github.com/kiegroup/drools-wb/pull/1214
https://github.com/kiegroup/appformer/pull/767
https://github.com/kiegroup/kie-wb-common/pull/2815

### CODE REVIEW

**Comment by danielzhe** [comment]
@karreiro @jomarko Can you check it, please?

**Comment by jomarko** [comment]
@danielzhe Hi, I will provide final feedback here. I haven't commented code of first two parts of ensemble. However this last PR gave me some ideas about whole ensemble.

- One issue is that the highlight doesn't work in the DMN grids. 
- The second point in my mind is, we introduce and observe an event (`SearchPerformedEvent`) that is not needed from my point of view. I think we just need to update `GridHighlightHelper#select` method to apply also the background color. I think this because @karreiro introduced concept of searchable elements. Each searchable element has callback `onFound` that is invoked in same situation, when we observe `SearchPerformedEvent`. All our searchable elements use the `GridHighlightHelper#select`, that is why I think we could try to update this method. What do you think?

**Comment by danielzhe** [comment]
@jomarko I would agree with you if we have more time, because to achieve this we would have to make changes in Lienzo level. About the DMN Grids, I'm checking.

**Comment by danielzhe** [comment]
@jomarko Changes applied! Is working on DMN Grids too. @karreiro

**Comment by jomarko** [comment]
@danielzhe thank you for updates, could you please rebase the ensemble? There are some conflicts now.

**Comment by danielzhe** [comment]
@karreiro @jomarko Ready for your re-review.

File: drools-wb-webapp/src/main/webapp/WEB-INF/beans.xml [hunk 1]
```diff
@@ -4,6 +4,12 @@
     <!-- These exclusions were added by Errai to avoid deploying client-side classes to the server -->
     <!-- End of Errai exclusions -->
 
+    <!-- These exclusions were added by Errai to avoid deploying client-side classes to the server -->
+    <!-- End of Errai exclusions -->
+
+    <!-- These exclusions were added by Errai to avoid deploying client-side classes to the server -->
+    <!-- End of Errai exclusions -->
+
```
Comment (jomarko) [inline drools-wb-webapp/src/main/webapp/WEB-INF/beans.xml]: Please do not include these changes in PR.

Comment (jomarko) [reply drools-wb-webapp/src/main/webapp/WEB-INF/beans.xml]: A reminder, we shouldn't these changes of `beans.xml` in this PR.

Comment (jomarko) [reply drools-wb-webapp/src/main/webapp/WEB-INF/beans.xml]: @danielzhe please, ensure no changes are done in beans.xml as part of drools-wb#1214 PR

**Pull request merged**